erpnext/hr/utils.py

```diff
@@ -49,3 +49,9 @@ def update_employee(employee, details, cancel=False):
 			new_data = get_datetime(new_data)
 		setattr(employee, item.fieldname, new_data)
 	return employee
+
+
+def get_leave_period(from_date, to_date, company):
```

Comment:
this missing function is defined in https://github.com/frappe/erpnext/pull/13951/files. Please copy it from there.